fixes.ini

```diff
@@ -1,7 +1,8 @@
 [Fixes]
 PatchInput = true
 PatchVirtualMemory = true
-PatchCalendarCrash = ture
+PatchCalendarCrash = true
+ShowXI4JWindow = true
```

Comment:
This window's initial purpose was to debug that the parch is loaded :) I think the default value sgould be `false`